PsfLauncher/main.cpp

```diff
@@ -51,6 +53,11 @@ int __stdcall wWinMain(_In_ HINSTANCE, _In_opt_ HINSTANCE, _In_ PWSTR args, _In_
 int launcher_main(PCWSTR args, int cmdShow) noexcept try
 {
     Log("\tIn Launcher_main()");
+
+    //Log("\tWaiting for debugger to attach...");

```

Comment:
Please remove commented out code if it is not needed.

Reply:
done.